src/config/config.model.ts

```diff
@@ -41,18 +43,23 @@ export const emptyConfig: Config = {
   cache: {
     evictionDelayHrs: 24,
   },
+  azure: {
+    parentManagementGroups: [],
+  },
 };
 
 function getConfigPath(): string {
   if (os.default.platform() === "windows") {
-    return Deno.env.get("%APPDATA%") + "/.config/meshcloudShepherd";
+    return Deno.env.get("%APPDATA%") + "/.config/meshcloud-collie";
   } else {
-    return Deno.env.get("HOME") + "/.config/meshcloudShepherd";
+    return Deno.env.get("HOME") + "/.config/meshcloud-collie";
```

Comment:
Let's just not mention `meshcloud` at all. I propose to name the folder `collie-cli`.

Reply:
Okay collie-cli there might be okay. I feared that we might run into a collision there.